### PR TITLE
Add note on return value coercion in error handler

### PR DESCRIPTION
**Pull request by othercorey**
Refs https://github.com/php/doc-en/issues/1206

@cmb69 Maybe this will help.

I don't know the standard wording for type declarations and types in the documentation.

### CODE REVIEW

File: reference/errorfunc/functions/set-error-handler.xml [hunk 1]
```diff
@@ -148,7 +148,8 @@
        </variablelist>
       </para>
       <para>
-       If the function returns &false; then the normal error handler continues.
+       If the function returns &false; then the normal error handler continues. The return value is not coerced to &bool;
```
Comment (saundefined) [inline reference/errorfunc/functions/set-error-handler.xml:151]: to fix the build:

```suggestion
       If the function returns &false; then the normal error handler continues. The return value is not coerced to &boolean;
```

Comment (othercorey) [reply reference/errorfunc/functions/set-error-handler.xml]: Thank you.

**Comment by othercorey** [comment]
Closing due to no interest.

**Comment by cmb69** [comment]
> Closing due to no interest.

There are still 75 open pull requests, many of which are lying around for much longer. This is not because there is no interest, but rather due to lack of time. :)

Anyhow, I think this patch wouldn't really cut it. There is a more general issue, namely that this callback (and others for other functions) do not have to use any type declarations at all, or might even have mismatching type declarations (https://3v4l.org/SochO#v7313; I'm slightly confused regarding the PHP 8.1 behavior), and we should probably document that in a central place, or at least use more general wording.

**Comment by othercorey** [comment]
> > Closing due to no interest.
> 
> There are still 75 open pull requests, many of which are lying around for much longer. This is not because there is no interest, but rather due to lack of time. :)

No worries. That was just meant to comment the closure not criticize it not being merged :). I noticed most disapproved PRs just stay open in php repos so that's all.

**Pull request closed without merging**